assets/web/chart.js

```diff
@@ -1,7 +1,16 @@
 /*!
- * Chart.js v2.9.4
- * https://www.chartjs.org
- * (c) 2020 Chart.js Contributors
- * Released under the MIT License
+ * Chart.js
+ * http://chartjs.org/
+ * Version: 2.4.0
```

Comment:
Isn't `2.4.0` older than `v2.9.4` ? Besides the last release is [v3.4.1](https://github.com/chartjs/Chart.js/releases). This [minified version](https://cdn.jsdelivr.net/npm/chart.js@3.4.1/dist/chart.min.js) mentions 3.4.1 too.